app/src/main/ui/src/i18n/de.json

```diff
@@ -138,6 +138,8 @@
   "usersImport.headlines.importErrors": "Fehler",
   "usersImport.headlines.importSuccess": "Import abgeschlossen",
   "usersImport.headlines.table": "Inhalt der ausgewählten Datei",
+  "usersImport.hint.newUser.summary":"Hinweis: Neu angelegte Anwender werden per E-Mail informiert",
```

Comment:
Wir haben zwar eine Details Komponente, aber könnte in die summary noch über was der Nutzer informiert wird stehen? "Login daten" oder "temporäres passwort", irgendwas in der richtung? Was denkst du? Passt das da rein oder sollten wir das lieber lassen?